base/doc/ltnews39.tex

```diff
@@ -144,7 +144,25 @@
 
 \section{Introduction}
 
-\emph{to write}
+The \LaTeX{} Project team remains strongly focused on producing automatically
+tagged PDF output for accessibility and reuse. At the beginning of
+2024 the ISO PDF/UA-2 and the WTPDF (well tagged PDF) standards have
+been released and we are glad to be able to report that it is now possible
+to automatically produce documents that conform to these new standards
+using \LaTeX.\footnote{At the present time we are still in a
+trial/prototype phase in which only a limited set of document classes and
+packages are supported.  Over the next releases we expect to gradually
+lift these restrictions and eventually provide the full functionality as
+part of the core distribution, rather than through \texttt{latex-lab}
+modules.}  A sample collection of such documents ranging from
+classical text, such as the bible, to modern papers, e.g., submitted to
```

Comment:
"Bible" is often capitalised? 

Not sure what a "modern paper" is? 

Maybe:
There is now a collection of sample documents 
ranging from classical texts such as the Bible to recent technical papers submitted to arXiv.org.
These can be found at . . .